addOns/automation/src/main/java/org/zaproxy/addon/automation/AuthenticationData.java

```diff
@@ -50,13 +51,20 @@
 import org.zaproxy.zap.utils.ZapXmlConfiguration;
 
 public class AuthenticationData extends AutomationData {
+    // TODO: Plan to change once the core supports dynamic methods better
+    protected static final String CLIENT_SCRIPT_BASED_AUTH_METHOD_CLASSNAME =
+            "org.zaproxy.addon.authhelper.client.ClientScriptBasedAuthenticationMethodType.ClientScriptBasedAuthenticationMethod";
+    protected static final String BROWSER_BASED_AUTH_METHOD_CLASSNAME =
+            "org.zaproxy.addon.authhelper.BrowserBasedAuthenticationMethodType.BrowserBasedAuthenticationMethod";
```

Comment:
Would be better to keep it in the same place, public has higher accessibility than protected.

Reply:
I think I misunderstood your earlier comment about moving it up then, or moving the comment up….

Reply:
Maybe the top of the protected grouping?